contrib/fuzz/cap_fuzzer.go

```diff
@@ -1,28 +0,0 @@
-//go:build gofuzz
-// +build gofuzz
-
-/*
-   Copyright The containerd Authors.
-   Licensed under the Apache License, Version 2.0 (the "License");
-   you may not use this file except in compliance with the License.
-   You may obtain a copy of the License at
-       http://www.apache.org/licenses/LICENSE-2.0
-   Unless required by applicable law or agreed to in writing, software
-   distributed under the License is distributed on an "AS IS" BASIS,
-   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-   See the License for the specific language governing permissions and
-   limitations under the License.
-*/
-
-package fuzz
-
-import (
-	"bytes"
-
-	"github.com/containerd/containerd/pkg/cap"
-)
-
-func FuzzParseProcPIDStatus(data []byte) int {
```

Comment:
Why was this removed?  Is it not needed for the oss-fuzz integration?

Reply:
This one is moved under pkg/cap.

Reply:
Yes, but the signature is now `func (*testing.F)` rather than `func ([]byte) int` and is moved into a `_test.go` file.  Does the external OSS-fuzz runner that consumes these functions (from https://github.com/containerd/containerd/pull/4841) know how to invoke a `func (*testing.F)`?

Reply:
> Does the external OSS-fuzz runner that consumes these functions (from #4841) know how to invoke a `func (*testing.F)`?


Yes. Some other changes might be needed in the build script, though. For example, utilities from `_test.go` files are currently not available to OSS-Fuzz, so these files need to be non `_test.go` files.

Reply:
> the signature is now func (*testing.F) rather than func ([]byte) int

Both can be used by OSS-Fuzz.